python-packages/contract_wrappers/src/zero_ex/contract_wrappers/_base_contract_wrapper.py

```diff
@@ -0,0 +1,125 @@
+"""Base wrapper class for accessing ethereum smart contracts."""
+from typing import Optional, Union
+from eth_utils import to_checksum_address
```

Comment:
Add a blank line here, between standard library imports and 3rd party imports.

Reply:
done.